changelog/dmd.named-arguments.dd

```diff
@@ -0,0 +1,32 @@
+Named arguments have been implemented
+
+In a function call expression, an argument can new be prefixed by the identifier of the parameter it binds to.
```

Comment:
Could you add a link to the DIP? That way, interested users will have more information.

@dkorpel I think we should also create a spec PR.

Reply:
I was going to add a spec PR and changelog (https://github.com/dkorpel/dmd/commit/8f72ce878127a2148634be666537066a90208e6b), but the implementation is unfinished (doesn't work with IFTI yet) so I planned to fix that first.